README.md

```diff
@@ -12,19 +12,13 @@ This is a Carbon Five-flavored convenience skeleton project for React. It is bas
 - Deployable to Heroku out of the box
 - Or deploy via docker using the included Dockerfile
 
-To get started, make sure you have Node 10+ and Yarn installed, and then generate your project like this:
+To get started, make sure you have Node 16 and Yarn installed, and then generate your project like this:
 
 ```
 $ npx spraygun -t react-ts <project-directory>
 ```
 
-If you'd like to use spraygun-react-ts with a Rails API backend, follow this guide:
-
-> [Using spraygun-react with a Rails backend](https://github.com/carbonfive/spraygun-react/blob/main/docs/how-to-use-with-rails-backend.md)
-
-If you'd like to use spraygun-react-ts with Redux, here are potential file structures:
-
-> [Redux file structures](https://github.com/carbonfive/spraygun-react/blob/main/docs/example-redux-file-structure.md)
```

Comment:
Should we also dump this doc page?  it doesn't seem very useful *even* if we kept the redux recommendation.

https://github.com/carbonfive/spraygun-react/blob/main/docs/example-redux-file-structure.md

Reply:
Yes, I am thinking about archiving that whole repo (spraygun-react) since all of my attention is now on the TypeScript version.